src/xb-builder.h

```diff
@@ -39,6 +39,7 @@ struct _XbBuilderClass {
  * @XB_BUILDER_COMPILE_FLAG_SINGLE_LANG:	Only store a single language
  * @XB_BUILDER_COMPILE_FLAG_WATCH_BLOB:		Watch the XMLB file for changes
  * @XB_BUILDER_COMPILE_FLAG_IGNORE_GUID:	Ignore the cache GUID value
+ * @XB_BUILDER_COMPILE_FLAG_SINGLE_ROOT:	Require a root node without siblings
```

Comment:
Perhaps “Require at most one root node”? Otherwise it’s ambiguous whether this results in an error if the document has no root nodes.

Reply:
Agreed, thanks. Fixed.